client/state/index.js

```diff
@@ -3,7 +3,6 @@
  */
 import thunkMiddleware from 'redux-thunk';
 import { createStore, applyMiddleware, combineReducers } from 'redux';
-
```

Comment:
Can we keep this newline for consistency's sake? Not sure if this was intentional.